Makefile

```diff
@@ -12,6 +12,10 @@ NODE_MODULES_TEST=package-lock.json
 # one example file in dist/ from webpack to check if that already ran
 WEBPACK_TEST=dist/index.html
 
+WEBLATE_REPO=cockpit-podman-weblate
+WEBLATE_REPO_URL=git@github.com:cockpit-project/cockpit-podman-weblate.git
```

Comment:
You could use `$(PACKAGE_NAME)-weblate` as project name? (Again, for generalizing to starter-kit)

Reply:
I would need to use `RPM_NAME`, since `PACKAGE_NAME` here is `podman`. But repo is `cockpit-podman`. In starter-kit, the `PACKAGE_NAME` is `starter-kit` and repo is also `starter-kit` and not `cockpit-starter-kit`. In `cockpit-composer` the `PACKAGE_NAME` is `cockpit-composer`, but there is different organization.
So either we would need to fix here what the `PACKAGE_NAME` is, but that would change other things as well (like name of `.pot` file, which would mean we would have to update weblate now...)
So it is all just mess and it is simpler to just specify the correct repo URL than trying to fit it in.

Reply:
Ack.